src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -485,65 +483,48 @@ public static BufferedImage getHideIcon(Color color)
-   * is loaded, and the Sprite is created, on the first call to this function, and simply
-   * returned thereafter. Letters are stored in order starting from 'A'.
-   * @return A Sprite object containing the in-game menu font for uppercase letters.
-   */
-  public static Sprite getLettersUppercase()
-  {
-    if( null == letterSpritesUppercase )
-    {
-      letterSpritesUppercase = new Sprite(SpriteLibrary.loadSpriteSheetFile("res/ui/main/letters_uppercase.png"), 5, 11);
-    }
-    return letterSpritesUppercase;
-  }
-
-  /**
-   * Returns a Sprite containing every lowercase letter, one letter per frame. The image
-   * is loaded, and the Sprite is created, on the first call to this function, and simply
-   * returned thereafter. Letters are stored in order starting from 'a'. 
-   * @return A Sprite object containing the in-game menu font for lowercase letters.
-   */
-  public static Sprite getLettersLowercase()
+  public static PixelFont getFontStandard()
   {
-    if( null == letterSpritesLowercase )
+    if( null == fontStandard )
     {
-      letterSpritesLowercase = new Sprite(SpriteLibrary.loadSpriteSheetFile("res/ui/main/letters_lowercase.png"), 5, 11);
-    }
-    return letterSpritesLowercase;
-  }
-
-  /**
-   * Returns a Sprite containing the digits 0-9, one number per frame. The image
-   * is loaded, and the Sprite is created, on the first call to this function, and simply
-   * returned thereafter. Numbers are stored in order starting from 0.
-   * @return A Sprite object containing the in-game menu font for numbers.
-   */
-  public static Sprite getNumbers()
-  {
-    if( null == numberSprites )
-    {
-      numberSprites = new Sprite(SpriteLibrary.loadSpriteSheetFile("res/ui/main/numbers.png"), 5, 11);
+      try
+      {
+        final File folder = new File("res/");
+        if( folder.canRead() )
+        {
+          for( final File fileEntry : folder.listFiles() )
+          {
+            String filepath = fileEntry.getAbsolutePath();
+            // Look for files with our extension
+            if( !fileEntry.isDirectory() && filepath.endsWith(".ttf") )
+            {
+              // The name should be formatted as fontname_wxh.ttf
+              String[] nameparts = fileEntry.getName().split("[_x.]");
+
+              if( nameparts.length != 4 ) continue;  // Not formatted correctly.
+
+              int w = Integer.parseInt(nameparts[1]);
+              int h = Integer.parseInt(nameparts[2]);
+
+              Font temp = Font.createFont(Font.TRUETYPE_FONT, fileEntry);
+              fontStandard = new PixelFont(temp, w, h);
+              System.out.println("Using font " + fileEntry.getName());
+              break;
+            }
+          }
+        }
+      }
+      catch(Exception ex)
+      {
+        System.out.println("WARNING! Exception loading font file. Using fallback");
+        ex.printStackTrace();
+        Font temp = new Font("Consolas", 0, 10);
+        fontStandard = new PixelFont(temp);
```

Comment:
This fallback doesn't trigger if there's just no valid font available.

Reply:
Huh. I somehow end up with a working font on Windows. But yeah, that logic is wrong. Fixed.